lib/ioki/model/platform/phone_verification.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Platform
+      class PhoneVerification < Base
+        attribute :phone_number, on: :create, type: :string
+        attribute :code, on: :create, type: :string
+      end
+    end
+  end
+end
```

Comment:
Have you seen #75? This introduces matchers to have at least some spec coverage for this. What do you think of this?

Reply:
Added specs as suggested!